### DISCUSSION TITLE
[Bug] Fix the visibility of the CDC log, it should only become visible after the KV data has been flushed to RocksDb

### DISCUSSION THREAD

**Discussion by swuferhong** [apache/fluss#81] [General]
### Search before asking

- [X] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.

### Fluss version

This issue is report bug that the visibility of the CDC log, it should only become visible after the KV data has been flushed to RocksDb. Currently, Fluss will first write the kv data to PreWriteBuffer in memory cache, and then flush to RocksDb, but in this time interval gap, the CDC log is visible to user, it's a wrong act.

### Minimal reproduce step

no reproduce step

### What doesn't meet your expectations?

In daily tests

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR!

**Comment by wuchong** [comment]
convert to issue #134
